framework/Samplers/Sampler.py

```diff
@@ -295,7 +295,7 @@ def _readMoreXMLbase(self,xmlNode):
           self.raiseAnError(IOError,'The following are duplicated variables indices listed in the manifestVariablesIndex: ' + str(dups))
         listElement = self.distributions2variablesMapping[dist]
         for var in listElement:
-          self.variables2distributionsMapping[var.keys()[0]]['totDim'] = maxDim #reset the totDim to reflect the totDim of original input space
+          self.variables2distributionsMapping[utils.first(var.keys())]['totDim'] = maxDim #reset the totDim to reflect the totDim of original input space
```

Comment:
Seems safer and also I like the fact that there is not a literal value sitting in the code. Something just feels better about this.